### DISCUSSION TITLE
How to Ctrl + C?

### DISCUSSION THREAD

**Discussion by andersoncampolina** [spyoungtech/ahk#162] [Q&A]
Hello folks!

Im tryin to execute this code:

> 
ahk = AHK()
time.sleep(1)
ahk.key_press('lwin') 
time.sleep(1)
ahk.type(r'C:\Users\zehn1984\Desktop\Fotos e videos sitio')
time.sleep(1)
ahk.key_press('enter')
time.sleep(1)
ahk.click(730, 474)
time.sleep(1)
ahk.key_down('Ctrl')
time.sleep(1)
ahk.key_press('c')
time.sleep(1)
ahk.key_up('Ctrl')
>

But it not working on the ctrl+c issue. Can someone help?

ps: theres a way to put a fixed pause time between each command, like in pyautogui (pyautogui.PAUSE = 1)?

**Comment by spyoungtech** [comment]
I think something like this should work
```python
ahk.send('{Ctrl down}{c down}')
```

**Reply by i448** [reply]
Trying to simulate press and hold, how to release? Like this?


`ahk.send_input("6{Blind Down}")`
`time.sleep(1)`
`ahk.key_release("6")`

**Reply by spyoungtech** [reply]
In v1 this should work:

```python
ahk.send("{6 DOWN}") # press and hold the key
for _ in range(3):
    if ahk.key_state('6'):
        print('Key is down')
        time.sleep(1)
ahk.key_up("6")
print('key is up' if not ahk.key_state('6') else 'key is not up')
```
